android/app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -118,6 +120,17 @@ private void handleVisibility() {
             locationsRecyclerView.setVisibility(View.GONE);
         }
     }
+    private void noResult() {
+        if (locationsListAdapter.getItemCount()!=0) {
+            noResultsView.setVisibility(View.GONE);
+            locationsRecyclerView.setVisibility(View.VISIBLE);
+        } else {
+            if (noMicrolocationsView.getVisibility() != View.VISIBLE) {
```

Comment:
Why are you putting it as else { if ( ) -- . Maybe you could write it as else if () { -- }